resources/assets/uad_lists.json

```
@@ -11421,6 +11421,14 @@
     "labels": [],
     "removal": "Recommended"
   },
+  "com.lge.lgdmsclient": {
+    "list": "Oem",
+    "description": "Actual "Software Update" App client\nI think you won't receive updates without this.\n",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Expert"
```
Why would you want to remove an application that keeps your phone up-to-date and thus secure?

---

Since LG shutdown its smartphone division there won't be updates anymore. So removing it just saves battery and also on my old lg v30 I randomly got installed few apps (sponsored apps/ads/more bloat) without my consent, after literally 3-4 years since the latest update. This happened few times actually.

---

I'd keep this to recommended honestly.

---

Even though you have a valid point here, there are most likely still people who aren't on the latest version yet. Those people are probably not users who use UAD-ng, but for the sake of consistency, it should be `Expert`. Users can still uninstall it. We can't (and shouldn't) treat software update packages different.